python-lib/xlsx_writer.py

```diff
@@ -8,16 +8,22 @@
 
 import logging
 import math
+import os
+import tempfile
+import zipfile
+
 from typing import Tuple, List, Dict
 from copy import copy
 
-from openpyxl.styles import Alignment, Font, PatternFill, Side
+from openpyxl.cell import Cell
+from openpyxl.styles import Alignment, Border, Fill, Font
```

Comment:
`Border` get redefined at the next line..

Reply:
done in 879e49419a0aaeb5b72f8b09fc085f83644ad80b